.teamcity/_self/lib/e2e/Steps.kt

```diff
@@ -44,3 +46,41 @@ fun BuildSteps.prepareEnvironment(): ScriptBuildStep {
 		dockerRunParameters = "-u %env.UID%"
 	}
 }
+
+/**
+ * Collects the resulting artifacts from a test run.
+ *
+ * This function will collect the following:
+ * 	- Screenshots
+ *	- Logs
+ *	- Trace files
+ */
+fun BuildSteps.collectResults(): ScriptBuildStep {
```

Comment:
You might be able to move this back into the body of the main class by using `bashNodeScript`. I think it sets nearly all of the script properties in the same way

Reply:
Sure. I had kept the `collectResults` step split out in case other classes may want to use it, but that doesn't appear to be the case for now.

Reply:
I ended up moving both `collectResults` and `prepareEnvironment` out to be defined in the class instead.